src/dialogs/generic/dialog-box.ts

```diff
@@ -76,7 +76,7 @@ class DialogBox extends LitElement {
                 <ha-textfield
                   dialogInitialFocus
                   .value=${this._value || ""}
-                  @keyup=${this._handleKeyUp}
+                  @keyup=${this._valueChanged}
                   @change=${this._valueChanged}
```

Comment:
```suggestion
                  @input=${this._valueChanged}
```

Reply:
This will update the changed value ,immediately

Reply:
Suggestions applied